test/other_unit_test.dart

```diff
@@ -75,5 +82,260 @@ main() {
 //    expect(design.helices[1].position3d().z, 20);
   });
 
+  group('strand_maker_tests', () {
+    test('test_strand__0_0_to_10_cross_1_to_5', () {
+      Design actual_design = new Design();
```

Comment:
Can you re-write these unit tests to work as in my examples, e.g. instead of

```dart
Design actual_design = new Design();
StrandMaker maker = new StrandMaker(0, 0);
maker.to(10);
maker.cross(1);
maker.to(5);
Strand new_strand = maker.commit();
actual_design = actual_design.rebuild((s) => s..strands.add(new_strand));
```

write 

```dart
Design actual_design = new Design(grid: Grid.square, helices: [Helix(idx: 0), Helix(idx:1)]);
actual_design = actual_design.strand(0, 0).to(10).cross(1).to(5).commit();
```

Note that you will have to put helices in the designfirst. There should be error-checking code to ensure that `Domain`'s refer to actual helices when the `Design` object is created, and it should throw an `IllegalDesignError` if you to put a `Strand` into a `Design` that references a `Helix` that the `Design` does not have.

Here, you are side-stepping that code by essentially setting the `strands` field directly, but the `commit()` method should check that, or alternately it could call a new method on `Design` called `add_strand` that does this check.